public/app/features/query/components/QueryGroupOptions.tsx

```diff
@@ -200,6 +218,14 @@ export class QueryGroupOptionsEditor extends PureComponent<Props, State> {
             <>
               <div className="gf-form-label query-segment-operator">=</div>
               <div className="gf-form-label">Width of panel</div>
+              <div className="gf-form-label query-segment-operator">*</div>
+              <Select
+                width={10}
+                value={options.resolution}
+                options={resolutionOptions}
+                placeholder="1"
```

Comment:
I'm not sure whether this placeholder is fine, maybe it should be something like '100%'?

Reply:
yea, true!